### PR TITLE
modified files in the plugins directory.

### LINKED ISSUE #13307: Incorrect file docstrings in tests/support/plugins directory
Whilst browsing through `tests/support/plugins/*.py` I noticed that some of the file docstrings are not correct, e.g. `selenium.py` has
https://github.com/bokeh/bokeh/blob/38d05d037223234aedeb25e35502abf3332bf0b4/tests/support/plugins/selenium.py#L7

which looks like a copy&paste of `log_file.py`.

All of the files in that directory should be checked at the same time in case others need updating.

### PR DESCRIPTION
**Pull request by Azaya89**
- [x] issues: fixes #13307 
- [x] tests added / passed

### CODE REVIEW

**Comment by bryevdv** [comment]
@Azaya89 the changes to `bokehjs/package-lock.json` are unrelated and incidental, can you remove the changes to that file from this PR?

File: tests/support/plugins/file_server.py
Comment (bryevdv) [inline tests/support/plugins/file_server.py:9]:
```suggestion
Used serving the test html pages that are needed by the webdriver unit
```
I think this was the intended meaning

Comment (bryevdv) [inline tests/support/plugins/file_server.py]:
```suggestion
Serve html pages that are needed by the webdriver unit tests.
```

I think this was the intended meaning(also, delete the "tests." on the following line)

**Review by ianthomas23 — APPROVED** [review]
Looks good to me. I'll wait for @bryevdv's approval too given his earlier review.

**Review by bryevdv — APPROVED** [review]

**Comment by ianthomas23** [comment]
Thanks @Azaya89!

**Pull request merged**
